styles/main-testing.css

```diff
@@ -0,0 +1,297 @@
+    }
+    .about-img img{
+        width: 80%;
+    }
+    .about-text{
+    }
+    .about-text-para{
+        width: 80%;
+    }
+    .dog-bio *{
+        width: 80%;
+    }
+    .dog-bio{
+        margin: 5%;
+    }
+    .adopt-btn{
+        width: 80%;
+        color: white;
+        background-color: limegreen;
+        border-color: limegreen;
+        font-size: .75rem;
+        font-weight: bold;
+    }
+    .more-dogs-btn{
+        width: 80%;
+        color: darkslateblue;
+        background-color: white;
+        border-style: none;
+        font-size: 1.5rem;
+        font-weight: bold;
+    }
+    #bottom-banner{
+        display: grid;
+        background-color: darkslategrey;
+        text-align: center;
+    }
+    #bottom-banner *{
+        width: 80%;
+    }
+    #logo-bot{
+        margin-top: 10%;
+    }
+    .footer-text{
+        color: white;
+    }
+    #bot-nav{
+    }
+    #bot-nav li{
+        display: inline;
+        border-left: 2px solid white;
+    }
+    #bot-nav li.first{
+        border-left: none; padding-left: none;
+    }
+    #bot-nav li a{
+        text-decoration: none;
+        color: white;
+        font-size: 1rem;
+    }
+    #sorry{
+        display: none;
+    }
+}
+
+@media only screen and (min-width: 768px){
+    #top-banner{
+        display: grid;
+        grid-template-columns: 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33%;
+        grid-template-rows: 80% 20%;
+        background: url(../images/header-image.jpg) no-repeat center top;
+        background-size: cover;
+        }      
+    /*The grid-template-columns property can also be used to specify the size (width) of the columns.*/
+
+    #logo-top p{
+        font-size: 1rem;
+        color: limegreen;
+        grid-column: 2 / 6;
+        margin-top: -4%;
+    }
+
+    #logo-top{
+        grid-column-start: 2;
+        grid-column-end: 6;
+        text-align: center;
+        align-content: center;
+    }
+    #cart-summary{
+        grid-column-start: 10;
+        grid-column-end: 11;
+        grid-row-start: 1;
+        grid-row-end: 3;
+        display: grid;
+        grid-template-columns: 50% 50%;
+        color: white;
+        align-content: end;
+    }
+    .cart-summary-obj2{
+        width: 33%;
+    }
+    .cart-summary-obj3{
+        grid-row-start: 2;
+        grid-column-start: 1;
+        grid-column-end: 3;
+    }
+    .checkout-btn{
+        width: 100%;
+        color: white;
+        background-color: limegreen;
+        border-color: limegreen;
+        font-size: .75rem;
+        font-weight: bold;
+    }
+    #top-nav{
+        grid-column: 1 / 13;
+        grid-row: 3 / 4;
+        background: white;
+        color: darkslateblue
+    }
+    #top-nav li{
+        display: inline;
+        border-left: 2px solid darkslateblue;
+        padding-left: 0.3em;
+    }
+    #top-nav li.first{
+        border-left: none; 
+        padding-left: none;
+    }
+
+    #main-content{
```

Comment:
It's not good practice to use id's for CSS. You could change these to classes, even if you only use the class once it is strongly preferred.